specificatie/genereervariant/openapi.json

```diff
@@ -2562,6 +2562,17 @@
         } ]
       },
       "Naam" : {
+        "allOf" : [ {
```

Comment:
Is deze bedoeld in deze fix ? De omschrijving doelt alleen op het burgerservicenummer example.

Reply:
Door deze fout is voor p.r. #851 geen genereervariant gegenereerd. Die generatie krijgt je nu daardoor in deze p.r.